Assets/Player.cs

```diff
@@ -31,7 +31,7 @@ public void Start()
 
     public void Update()
     {
-     
+        speed += 0.01f;
```

Comment:
Jakbyś mnie nie oznaczył to bym nie widział 😅

Jakiś limit by się przydał, żeby gracz zawsze miał możliwość zmiany pozycji pomiędzy przeszkodami.